ppr-ui/src/assets/styles/base.scss

```diff
@@ -890,4 +912,29 @@ tr.registration-row.draft-registration-row.added-reg-effect td.actions-cell,
 
 .overlap {
   z-index: 7;
+}
+
+// Contact Type Radio Btns
+.person-radio {
+  width: 47%;
```

Comment:
Another thing to cleanup (after design system in place) is these relative widths and use flexbox.